src/main/java/io/gravitee/policy/json2json/configuration/JsonToJsonTransformationPolicyConfiguration.java

```diff
@@ -16,18 +16,26 @@
 package io.gravitee.policy.json2json.configuration;
 
 import io.gravitee.policy.api.PolicyConfiguration;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
 
 /**
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
 public class JsonToJsonTransformationPolicyConfiguration implements PolicyConfiguration {
 
+    @Builder.Default
     private PolicyScope scope = PolicyScope.REQUEST;
 
     private String specification;
 
     //by default, we override the content-type to be backward compatible
+    @Builder.Default
     private boolean overrideContentType = true;
 
     public PolicyScope getScope() {
```

Comment:
You could also add @Getter and @Setter annotation.

Reply:
:javoue: 🙈